src/test/resources/TestNG.xml

```diff
@@ -9,10 +9,11 @@
             <class name="LoginTests"/>
             <class name="Homework15"/>
             <class name="Homework16"/>
-            <class name="homework17"/>
+            <class name="Homework17"/>
             <class name="Homework18"/>
             <class name="Homework19"/>
             <class name="Homework20"/>
+            <class name="Homework21"/>
```

Comment:
_Let’s execute TestNG classes in parallel using TestNG xml as below:_

Add `parallel="methods" thread-count="4"`

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG" parallel="methods" thread-count="4">
    <test name="Example Test" preserve-order="false">
        <parameter name="baseURL" value="https://qa.koel.app/"/>
        <classes>
            <class name="PlaylistTests"/>
            <class name="LoginTests"/>
        </classes>
    </test>
</suite>
```